test/ex_doc/autolink_test.exs

```diff
@@ -190,7 +236,13 @@ defmodule ExDoc.AutolinkTest do
 
       assert_unchanged("mix compile.elixir --verbose")
 
-      assert_unchanged("mix unknown.task")
+      captured =
+        assert_warn(fn ->
```

Comment:
same as above, if we expect not to warn let’s remove assert_warn.

Reply:
done